tests/test_loggers/test_console_logger.py

```diff
@@ -16,20 +17,50 @@
 from etna.transforms import AddConstTransform
 from etna.transforms import DateFlagsTransform
 from etna.transforms import LagTransform
+from etna.transforms import Transform
+
+
+def check_logged_transforms(log_file: str, transforms: Sequence[Transform]):
+    """Check that transforms are logged into the file."""
+    with open(log_file, "r") as in_file:
+        lines = in_file.readlines()
+        assert len(lines) == len(transforms)
+        for line, transform in zip(lines, transforms):
+            assert transform.__class__.__name__ in line
+
+
+def test_tsdataset_transform_logging(example_tsds: TSDataset):
+    """Check working of logging inside `TSDataset.transform`."""
+    transforms = [LagTransform(lags=5, in_column="target"), AddConstTransform(value=5, in_column="target")]
+    file = NamedTemporaryFile()
+    _logger.add(file.name)
+    example_tsds.fit_transform(transforms=transforms)
+    idx = tslogger.add(ConsoleLogger())
```

Comment:
why do you use `ConsoleLogger` with added file instead of `LocalFileLogger`?

Reply:
1. These tests were created before `LocalFileLogger`.
2. `LocalFileLogger` don't do anything on `log` method, it can only log backtest metrics.